common/src/main/java/com/github/teamfusion/summonerscrolls/common/registry/SummonerEvents.java

```diff
@@ -46,8 +40,8 @@ public static void useScroll(Player player, InteractionHand hand) {
             int damage = ScrollUtil.getDamage(nbt);
 
 
-            if (player.getCooldowns().isOnCooldown(item)) {
-                float cooldownTicks = player.getCooldowns().getCooldownPercent(item, 0);
+            if (((SummonerPlayerMixinHolder) player).summonerscrolls$getSummonerCooldowns().isOnCooldown(itemStack)) {
+                float cooldownTicks = ((SummonerPlayerMixinHolder) player).summonerscrolls$getSummonerCooldowns().getCooldownPercent(itemStack, 0);
```

Comment:
Just get the cooldowns once instead of writing sphagetti code.
```java
summonerCooldowns = ((SummonerPlayerMixinHolder) player).summonerscrolls$getSummonerCooldowns();
if (summonerCooldowns.isOnCooldown(itemStack)) {
  float cooldownTicks = summonerCooldowns.getCooldownPercent(itemStack, 0);
  // ...
```